client/src/modules/Job/pages/NoAuthLanding/components/TopSlide.tsx

```diff
@@ -0,0 +1,134 @@
+import { COLORS } from 'modules/Job/theme/colors';
+import { DottedRectangle } from './DottedRectangle';
+
+export function TopSlide() {
+  return (
+    <>
+      <div className="wrapper">
```

Comment:
Same as above regarding usage of Antd components instead of raw html and styles

Reply:
the same as above